shoplist/src/main/res/layout/main_fragment.xml

```diff
@@ -33,10 +33,10 @@
         app:layout_constraintTop_toTopOf="parent" />
 
     <androidx.recyclerview.widget.RecyclerView
-        android:layout_width="wrap_content"
-        android:layout_height="match_parent"
+        android:layout_width="match_parent"
```

Comment:
i took a look on the xml and found it's not arranged properly the list above the header so updated it to make views stable and avoid any ui/ux issues